packages/wdio-cli/src/utils.ts

```diff
@@ -287,8 +295,18 @@ export function getCapabilities(arg: ReplCommandArguments) {
         return { capabilities: { browserName: 'Chrome', ...ANDROID_CONFIG, ...optionalCapabilites } }
     } else if (/ios/.test(arg.option)) {
         return { capabilities: { browserName: 'Safari', ...IOS_CONFIG, ...optionalCapabilites } }
+    } else if (/(js|ts)$/.test(arg.option)) {
+        const config = new ConfigParser()
+        config.addConfigFile(arg.option)
+        let requiredCaps = config.getCapabilities()
+        if (Object.prototype.hasOwnProperty.call(arg, 'capabilities')) {
+            requiredCaps = (requiredCaps as (DesiredCapabilities | W3CCapabilities)[])[Number(arg.capabilities)] ||
+                (requiredCaps as MultiRemoteCapabilities)[arg.capabilities]
+        }
+        const requiredW3CCaps = pickBy(requiredCaps, (_, key) => !IGNORED_CAPABILITIES.includes(key))
```

Comment:
For which options are you looking here?

Reply:
@christian-bromann  maxInstance , specs , etc . These can be passed as desired capability but are not valid w3c

Reply:
Can we use this object: https://github.com/webdriverio/webdriverio/blob/main/packages/webdriver/src/constants.ts#L143-L147 ?

Reply:
@christian-bromann  We have vendor specific capabilities like "goog:chromeoptions" etc

Reply:
So we are removing unwanted properties and allowing all other capabilities

Reply:
Let's update the `pickBy` filter to allow only the valid WebDriver capabilities + the ones that have a colon (":"), we can remove the rest

Reply:
I am suggestion to reuse the `webdriver` constant because creating a new list of capabilities here makes it more difficult to maintain this code because, e.g. if we add a new option we need to update that list and all other lists too, so having a minimal amount of lists would be desireable.

Reply:
@christian-bromann 
![image](https://user-images.githubusercontent.com/45095911/167844975-c5f0e8b2-eba4-45a5-8a32-6a91e7bf8b88.png)

Reply:
I think it makes sense to move `VALID_CAPS` into `wdio-protocols` export it there and import it from the CLI and make `@wdio/protocols` a peer dependency.

Note: you can't do relative imports across package boundaries because when these packages are installed through NPM the folder structure might differ.

Reply:
@christian-bromann  i tried adding it to to the index file of protol package and adding it in depenedency but was showing no export member . COuld you show one example how to do this or which file to add this